### DISCUSSION TITLE
the crio.conf is empty

### DISCUSSION THREAD

**Discussion by stanleytho** [cri-o/cri-o#5689] [Q&A] [answered]
Hello, 

I downloaded the crio v1.23.1 and the default crio.conf in etc is empty. Is that intentional?

Thanks

**Comment by haircommander** [comment, marked as answer]
hey! yes see https://github.com/cri-o/cri-o/issues/5631. We have a backport in place for 1.23 https://github.com/cri-o/cri-o/pull/5570 but have yet to merge it
